lib/sidecloq/web.rb

```diff
@@ -6,7 +6,15 @@ module Web
 
     def self.registered(app)
       app.get '/recurring' do
-        @schedule = Schedule.from_redis
+        @job_specs = Schedule.from_redis.job_specs
+        @job_specs.each_value do |job_spec|
+          job_spec['cron'] ||= job_spec['every']
+
+          job_spec['queue'] ||= begin
```

Comment:
Possibly?  However, this code explicitly calls from_redis, so it appears to be redis specific to begin with (not sure why, just noting the observation)

Reply:
yeah, true.  but makes me think this logic should be in the schedule class... iow -- there seems to be some manipulation of the state of the schedule before display, such as the queue, so that logic either is duplicated somewhere or what's being displayed isn't matching what's actually being done (like the queue).  (one other option is that i just haven't had enough ☕ )

Reply:
I think the change to refactor into the `from_hash` method might be a bit out of my depth as I'm not as familiar with the code base.  I'm wondering if it makes sense to merge this PR as is, since it fixes the problem, and then a follow up could be done to refactor into `from_hash`